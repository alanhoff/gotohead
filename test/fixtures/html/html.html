<!DOCTYPE html>
<html>
   <head>
      <title>Go to Head</title>
      <style type="text/css"></style>
      <style data-above-the-fold="true"></style>
   </head>
   <body>
      <h1>Grunt plugin gotohead</h1>
   </body>
</html>
