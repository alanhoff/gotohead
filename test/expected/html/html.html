<!DOCTYPE html>
<html>
   <head>
      <title>Go to Head</title>
      <style type="text/css"></style>
      <style data-above-the-fold="true">body{font-size:100%}h1{font-size:2em}</style>
   </head>
   <body>
      <h1>Grunt plugin gotohead</h1>
   </body>
</html>
